force-app/main/default/classes/AccountTriggerHandler.cls

```diff
@@ -0,0 +1,72 @@
+public with sharing class AccountTriggerHandler {
+    public static void setTypeProspectAfterInsert(List<Account> newAccounts) {
+        List<Account> accountsToUpdate = new List<Account>();
+        //System.debug('number of new accounts: ' + newAccounts.size());
+        for (Account acc : newAccounts) {
```

Comment:
Method itself looks good, but remember you do not have access to edit Trigger.new